components/director/cmd/systemfetcher/main.go

```diff
@@ -716,3 +749,19 @@ func createSelectFilter(selectFilterProperties map[string]bool, placeholdersMapp
 
 	return selectFilter
 }
+
+func calculateSortedTemplateMappingKeys() {
+	templateMappingKeys := make([]systemfetcher.TemplateMappingKey, 0, len(systemfetcher.ApplicationTemplates))
+	for key := range systemfetcher.ApplicationTemplates {
+		templateMappingKeys = append(templateMappingKeys, key)
+	}
+
+	sort.Slice(templateMappingKeys, func(i, j int) bool {
+		if templateMappingKeys[i].Label != templateMappingKeys[j].Label {
```

Comment:
Here can be returned templateMappingKeys[i].Label < templateMappingKeys[j].Label directly without `if` clause, as it is a comparator function.